services/core/shared/core/accounts.js

```diff
@@ -28,7 +28,7 @@ const getAccounts = async params => {
 	const { status, ...remainingParams } = params;
 	let response = await coreApi.getAccounts(remainingParams);
 	if (status && ['active', 'banned', 'punished', 'standby'].some(item => status.includes(item))) {
-		response = await getDelegates(params);
+		response = params.isDelegate !== false ? await getDelegates(params) : { data: [] };
```

Comment:
Please consider adding the following comment.
```suggestion
		// Unless the request explicitly ignores delegates, consider all accounts call to include delegate info
		response = params.isDelegate !== false ? await getDelegates(params) : { data: [] };
```